src/components/Facepile/Facepile.Props.ts

```diff
@@ -8,6 +9,18 @@ export interface IFacepileProps extends React.Props<Facepile> {
    * is fixed at PersonaSize.extraSmall regardless of what is specified.
    */
   personas: IFacepilePersona[];
+
+  /** Maximum number of personas to show */
+  maxDisplayablePersonas?: number;
+
+  /** Show add person button */
+  showAddButton?: boolean;
+
+  /** Button properties for the add face button */
+  addButtonProps?: IButtonProps;
+
+  /** Properties for the overflow persona */
+  overflowPersonaProps?: IFacepilePersona;
```

Comment:
This is a Persona control at this time.  I can re-work it to be more of a button control if we really need more button level customization than when we get this way.